api/organisations/tasks.py

```diff
@@ -73,3 +86,103 @@ def finish_subscription_cancellation():
     ):
         subscription.organisation.cancel_users()
         subscription.save_as_free_subscription()
+
+
+def send_admin_api_usage_notification(
+    organisation: Organisation, matched_threshold: int
+) -> None:
+    """
+    Send notification to admins that the API has breached a threshold.
+    """
+
+    if matched_threshold < 100:
+        message = "organisations/api_usage_notification.txt"
+        html_message = "organisations/api_usage_notification.html"
+
+        # Since threshold < 100 only include admins.
+        recipient_list = list(
+            FFAdminUser.objects.filter(
+                userorganisation__organisation=organisation,
+                userorganisation__role=OrganisationRole.ADMIN,
+            ).values_list("email", flat=True)
+        )
+    else:
+        message = "organisations/api_usage_notification_limit.txt"
+        html_message = "organisations/api_usage_notification_limit.html"
+
+        # Since threshold >= 100 include everyone in the organisation.
+        recipient_list = list(
+            FFAdminUser.objects.filter(
+                userorganisation__organisation=organisation,
+            ).values_list("email", flat=True)
+        )
```

Comment:
I'd probably have done: 

```suggestion
    recipient_list = FFAdminUser.objects.filter(
        userorganisation__organisation=organisation,
    )

    if matched_threshold < 100:
        message = "organisations/api_usage_notification.txt"
        html_message = "organisations/api_usage_notification.html"

        # Since threshold < 100 only include admins.
        recipient_list = recipient_list.filter(
            userorganisation__role=OrganisationRole.ADMIN,
        )
    else:
        message = "organisations/api_usage_notification_limit.txt"
        html_message = "organisations/api_usage_notification_limit.html"
```

... and then in the send_mail call you can convert to a flat list of emails: 

```python
send_mail(
    ...
    recipient_list=list(recipient_list.values_list("email", flat=True))
```

Reply:
Ok that's no problem to update.